apps/ledger-live-mobile/src/screens/SendFunds/04-Summary.tsx

```diff
@@ -260,6 +274,14 @@ function SendSummary({ navigation, route }: Props) {
           navigation={navigation}
           route={route}
         />
+
+        {currency.id === "ethereum" && currency.type === "CryptoCurrency" && (
```

Comment:
I'm not sure we're aiming at only supporting ethereum, but even if it is, we should always avoid some "if ethereum" in the UI. When possible, centralize this behing either an helper or a simple env variable.

Reply:
also, we should avoid 
currency.type === "CryptoCurrency"
because we want to adapt this feature for erc20 tokens.